tests/test_filepath.py

```diff
@@ -56,14 +57,13 @@ def test_filepath_s3():
 def test_duplicate_upload(store="repo"):
     ext = schema.external[store]
     stage_path = dj.config['stores'][store]['stage']
-    filename = 'plot.dat'
     relpath = 'one/two/three'
-    os.makedirs(os.path.join(stage_path, relpath), exist_ok=True)
-    managed_file = os.path.join(stage_path, relpath, filename)
-    with open(managed_file, 'wb') as f:
+    managed_file = Path(stage_path, relpath, 'plot.dat')
```

Comment:
This test failed on WIN10. For example, the result of `managed_file` was `dj/repo/one\two\three\plot.dat`.